### PR TITLE
a few more things when reviewing replicated event sourcing

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: akka-docs/src/main/paradox/typed/guide/modules.md [hunk 1]
```diff
@@ -21,12 +21,10 @@ With a [Lightbend Platform Subscription](https://www.lightbend.com/lightbend-sub
 * [Configuration Checker](https://doc.akka.io/docs/akka-enhancements/current/config-checker.html) &#8212; Checks for potential configuration issues and logs suggestions.
 * [Diagnostics Recorder](https://doc.akka.io/docs/akka-enhancements/current/diagnostics-recorder.html) &#8212; Captures configuration and system information in a format that makes it easy to troubleshoot issues during development and production.
 * [Thread Starvation Detector](https://doc.akka.io/docs/akka-enhancements/current/starvation-detector.html) &#8212; Monitors an Akka system dispatcher and logs warnings if it becomes unresponsive.
-* [Kubernetes Lease](https://doc.akka.io/docs/akka-enhancements/current/kubernetes-lease.html) &#8212; Monitors an Akka system dispatcher and logs warnings if it becomes unresponsive.
```
Comment (chbatey) [inline akka-docs/src/main/paradox/typed/guide/modules.md:24]: ?

File: akka-persistence-typed/src/main/resources/reference.conf [hunk 2]
```diff
@@ -2,16 +2,16 @@ akka.actor {
 
   serialization-identifiers."akka.persistence.typed.serialization.ReplicatedEventSourcingSerializer" = 40
 
-  serializers.active-active = "akka.persistence.typed.serialization.ReplicatedEventSourcingSerializer"
+  serializers.replicated-event-sourcing = "akka.persistence.typed.serialization.ReplicatedEventSourcingSerializer"
```
Comment (chbatey) [inline akka-persistence-typed/src/main/resources/reference.conf:5]: 👍

**Review by chbatey — APPROVED** [review]
LGTM, will probably conflict with the serialization PR as i did some of the cleanups in that too

File: akka-docs/src/main/paradox/typed/guide/modules.md (hunk 1 above)
Comment (patriknw) [reply akka-docs/src/main/paradox/typed/guide/modules.md:24]: it's in Akka Management now, and this is a list of things in enhancement
(and the description was completely wrong)

**Pull request merged**